libs/aws/tests/integration_tests/chat_models/test_sagemaker_endpoint.py

```diff
@@ -130,3 +178,23 @@ def test_sagemaker_endpoint_invoke() -> None:
 
     assert service_response.content == "SageMaker Endpoint"
     assert isinstance(service_response, AIMessage)
+
+
+def test_sagemaker_endpoint_stream() -> None:
+    client = Mock()
+
+    stream_response = {"Body": MockStreamingBody()}
+    client.invoke_endpoint_with_response_stream.return_value = stream_response
+
+    chat_model = ChatSagemakerEndpoint(
+        endpoint_name="test-endpoint",
+        region_name="us-west-2",
+        content_handler=ContentHandlerStream(),
+        streaming=True,
```

Comment:
Is this necessary for streaming to work?

Reply:
It is not. Removed